docs/notes-pa.md

```diff
@@ -3,4 +3,166 @@ how the permutation argument is formed.
 
 
 Permutation argument
-==============
\ No newline at end of file
+==============
+
+Within PLONK, the permutation 
+polynomials are used to compare 
+arguments about the proving 
+scheme.  
+
+These arguments are 
+checking that wire values, 
+within the circuit, are
+being correctly copied. 
+As is noted in other parts
+of the docs, there is the 
+need for a copy check in 
+in SNARK schemes. This is 
+crucial in our zk-SNARK
+scheme, to ensure the 
+correctness of wires when 
+bundling similar wires, 
+and for ensuring values are
+travelling the correct way
+along wires.
+
+PLONK uses an argument to check 
+the correctness of a shuffle in 
+its proving algorithm. This is 
+essentially showin that the 
+products of polynomials are 
+the same, despite the ordering 
+being constructed differently.
+In principle, PLONK can compare sets 
+of polynomials for equality by 
+comparing `n` number of polynomials 
+with the same `n` number of polynomials 
+, where the elements have been 
+shuffled. 
+
+Given a set of  values, at a 
+particular position, we can create 
+a mapping for the initial index into a 
+new position, with a permutation. 
+
+For example: 
+Given initial values, 
+1,2,3,4,5 each occupying 
+their respective positions
+A,B,C,D,E
+
+\begin{aligned}
+
+// +-----+-----+-----+-----+-----+
+// |  A  |  B  |  C  |  D  |  E  |
+// +-----+-----+-----+-----+-----+
+// |  1  |  2  |  3  |  4  |  5  |
+// +-----+-----+-----+-----+-----+
+
+\end{aligned}
+
+If we then have a mapping of:
+
+\begin{aligned}
+
+\\[
+\mathbf{a}(1) = 5
+\mathbf{a}(2) = 2
+\mathbf{a}(3) = 3
+\mathbf{a}(4) = 1
+\mathbf{a}(5) = 4
+\\]
+
+\end{aligned}
+
+Which gives a second, permutuated row, 
+of:
+
+\begin{aligned}
+
+// +-----+-----+-----+-----+-----+
+// |  A  |  B  |  C  |  D  |  E  |
+// +-----+-----+-----+-----+-----+
+// |  1  |  2  |  3  |  4  |  5  |
+// +-----+-----+-----+-----+-----+
+// |  3  |  5  |  1  |  4  |  2  |
+// +-----+-----+-----+-----+-----+
+
+\end{aligned}
+
+We can form equations of polynomials 
+to compare the wire values at each 
+position. As it possible to falsify
+proofs, by having the sum of the 
+equation arguments equal - without 
+having the identities between the
+monomials hold - therefore, two random 
+challenges are introduced to the 
+equation. 
+
+The first is Beta, which is mutiplied
+by the index and the second is 
+gamma. Both of which, are taken from 
+the prime field. 
+
+To isolate the terms, for better evaluation,
+we are also capable of describing each wire
+as a type. For this example, we will say 
+wires A & B = type 1\\, wires C & D = type 2\\
+and wire E = type 3.
```

Comment:
I'm not sure I understand this section.

- For my understanding, could you explain this part "_To isolate the terms, for better evaluation, we are also capable of describing each wire as a type._" It sort of reads like a developer comment, or just me?
- wires A & B means concatenating the wires? 
- In the latter section when you write 1 + beta + gamma, does the 1 mean type 1?

Reply:
- 'For my understanding, could you explain this part "To isolate the terms, for better evaluation, we are also capable of describing each wire as a type." It sort of reads like a developer comment, or just me?' - Well I used types instead of colour blocks which are the norm for the explanation but is hard to show in a description like this. 
- 'wires A & B means concatenating the wires?' - Nope, could be any wire separate wires. As in the structure I showed above 
- 'In the latter section when you write 1 + beta + gamma, does the 1 mean type 1?' - Yes, I mean type 1 here